src/ShopBundle/objects/db/TShopOrder.class.php

```diff
@@ -488,6 +488,11 @@ public function MarkOrderAsExportedToWaWi($bExported = true)
      */
     public function SendOrderNotification($sSendToMail = null, $sSendToName = null)
     {
+        if ('1' === $this->sqlData['system_order_notification_send']) {
```

Comment:
The property system_order_notification_send ist not set at this point. I don't know why.
Needs to check why the property is not set and/or use $this->fieldSystemOrderNotificationSend